ops/model.py

```diff
@@ -1244,15 +1250,18 @@ class Relation:
             For subordinate relations, this set will include only one unit: the principal unit.
         data: A :class:`RelationData` holding the data buckets for each entity
             of a relation. Accessed via eg Relation.data[unit]['foo']
+        active: Indicates whether this relation is active (True) or inactive (False, because
+            this relation is currently part of a relation-broken event)
```

Comment:
These attributes have now been type-hinted and their docstrings moved to under the type hint so Sphinx picks it up for each attribute, so I believe this will conflict with the `main` branch now. Let's do something like this:

```python
class Relation:
    ...

    active: bool
    """Indicates whether this relation is active.

    This will be ``False`` if the current event is a ``relation-broken`` event
    associated with this relation.
    """
```